metis-core/metis-core-common/src/main/java/eu/europeana/metis/core/rest/execution/details/PluginProgressView.java

```diff
@@ -0,0 +1,41 @@
+package eu.europeana.metis.core.rest.execution.details;
+
+import eu.europeana.cloud.common.model.dps.TaskState;
+import eu.europeana.metis.core.workflow.plugins.ExecutionProgress;
+
+public class PluginProgressView {
```

Comment:
I think we can have this class declared inside the PluginView?
What do you think?

Reply:
In the overview package the equivalent class is also separated in its own file. Perhaps just keep it consistent with that?

Reply:
Then fine for now but I think we need to rethink the names of those two classes..
Having the same name in two different packages can be confusing.